packages/esm-patient-medications-app/src/add-drug-order/drug-order-form.component.tsx

```diff
@@ -302,9 +302,15 @@ export function DrugOrderForm({ initialOrderBasketItem, onSave, onCancel, prompt
     [orderConfigObject, config?.daysDurationUnit],
   );
 
-  const orderFrequencies: Array<MedicationFrequency> = useMemo(
-    () => orderConfigObject?.orderFrequencies ?? [],
-    [orderConfigObject],
+  const orderFrequencies: Array<MedicationFrequency> = useMemo(() => {
+    return orderConfigObject?.orderFrequencies ?? [];
+  }, [orderConfigObject]);
+
+  const filterItems = useMemo(
+    () => (menu) => {
+      return menu?.item?.names?.some((abbr) => abbr.toLowerCase().includes(menu?.inputValue?.toLowerCase()));
+    },
+    [],
```

Comment:
Since this is wrapping a function, its preferable to use `useCallback()` instead of `useMemo()`.

I'd probably also do something like:

```
menu?.inputValue?.length ? menu.item?.names?.some((abbr) => abbr.toLowerCase().includes(menu.inputValue.toLowerCase())) : menu.item?.names ?? [];
```

And the complexity of that I think points to the fact that this shouldn't be a one-liner.